packages/ui/src/app-drawer/AppDrawer.tsx

```diff
@@ -113,21 +111,13 @@ export function AppDrawer({
         <DrawerActions>
           <StyledActionsLayout>
             {primaryAction && (
-              <Button
-                type="button"
-                onClick={primaryAction.onClick}
-                variant="contained"
-              >
+              <Button type="button" variant="contained" {...primaryAction}>
```

Comment:
@sanjar-sd Are the design team aware that the button variant will not be customizable?